### PR TITLE
Containerize SNCRS

### PR DESCRIPTION
**Pull request by Jyckle**
Time to bring SNCRS to the modern devops world!
Containers are everywhere, and I'm sick of my hacky deploys. This should help significantly! 
This fulfills issue #2 and also transitions from sqlite to postgres for ease of use in the container
I added the `qs` script and documentation in the readme to make getting spun up easier as well. I still need to do a bit of testing on windows with docker desktop to ensure it works similarly, but it should!

### CODE REVIEW

File: sncrs/data/templates/data/teams.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
-<table class="table table-borderless table-dark">
-  <tr>
+<table class="table table-borderless table-dark text-center">
```
Comment (coltatgh) [inline sncrs/data/templates/data/teams.html:1]: Do these html updates have to do with the containerization at all, or are these just unrelated aesthetic updates made on the side?

**Review by coltatgh — APPROVED** [review]
Reviewed largely for my own education, will likely ask for some summarizing next time we meet as well.

File: sncrs/data/templates/data/teams.html (hunk 1 above)
Comment (Jyckle) [reply sncrs/data/templates/data/teams.html:1]: Great question! The html updates are not related to the containerization, except in that I noticed that without any data in the database, the home page looked really odd. So this was to solve that issue for fresh installs!

**Comment by Jyckle** [comment]
> Reviewed largely for my own education, will likely ask for some summarizing next time we meet as well.

Sounds great!

**Pull request merged**